interfaces/quizoot.ts

```diff
@@ -245,12 +245,12 @@ export declare namespace Quizoot {
          * Id of the next question in the quiz chronology. Is null if it is the last question.
          * @nullable
          */
-        next_question_id: string;
+        next_question_id: string | null;
```

Comment:
now I wonder why we have `null` and not just `undefined` haha.

Reply:
I don't like having `undefined`. I feels like something went wrong 😂